src/transformStateWithClones.js

```diff
@@ -6,8 +7,53 @@
  *
  * @return {Object[]}
  */
+
+const addProperties = (actionExtraData, transformedState) => {
```

Comment:
Instead of using 'actionExtraData', consider renaming the parameter to 'extraData' for better readability.